master/internal/trial_workload_sequencer.go

```diff
@@ -29,8 +29,7 @@ type (
 		CurOpIdx  int `json:"cur_op_idx"`
 		CurStepID int `json:"cur_step_id"`
 
-		LatestCheckpoint *model.Checkpoint            `json:"latest_checkpoint"`
-		LatestSnapshot   *trialWorkloadSequencerState `json:"latest_snapshot"`
+		LatestSnapshot *trialWorkloadSequencerState `json:"latest_snapshot"`
```

Comment:
You're going to need to write a shim for this state change.

Reply:
Ohhh, I saw you didn't include the shimming code for `cached_checkpoint` so I thought it not necessary to do so. Am I misunderstanding your code?

Reply:
Ah I think I should've removed that too, oops. Do you mind adding it to my shim? You can piggy back this change into my shim soon since they're going to go in a release together.

Reply:
Okay. I'll include that too in this PR.